packages/atlas/src/components/_navigation/SidenavViewer/FollowedChannels.styles.ts

```diff
@@ -13,35 +13,60 @@ export const FollowedChannelsWrapper = styled.div`
 `
 
 export const ChannelsTitle = styled(Text)`
+  font: ${cVar('typographyDesktopH100')};
   margin-top: ${sizes(6)};
   margin-bottom: ${sizes(4)};
   padding-left: ${NAVBAR_LEFT_PADDING}px;
   width: ${EXPANDED_SIDENAVBAR_WIDTH - NAVBAR_LEFT_PADDING}px;
-  color: ${oldColors.gray[300]};
+  color: ${cVar('colorText')};
+  text-transform: uppercase;
+  letter-spacing: 0.07em;
 `
+
 export const ChannelsWrapper = styled.div`
-  padding-left: ${NAVBAR_LEFT_PADDING}px;
   width: ${EXPANDED_SIDENAVBAR_WIDTH}px;
   overflow-y: auto;
   overflow-x: hidden;
   margin-bottom: 10px;
 `
 
 export const ChannelsList = styled.ul`
-  width: ${EXPANDED_SIDENAVBAR_WIDTH - NAVBAR_LEFT_PADDING}px;
+  color: ${cVar('colorText')};
+  width: ${EXPANDED_SIDENAVBAR_WIDTH}px;
   overflow-x: hidden;
   padding: 0;
   margin: 0;
 `
-export const StyledChannelLink = styled(ChannelLink)`
+
+type HasSecondaryProps = {
+  textSecondary: boolean
+}
+
+export const StyledChannelLink = styled(ChannelLink)<HasSecondaryProps>`
```

Comment:
Why do we need `HasSecondaryProps` here?

Reply:
So I can pass the secondary props: 

```
                <StyledChannelLink id={id} textSecondary={true} onNotFound={() => onChannelNotFound?.(id)} />
```

Reply:
But this is type is defined in `ChannelLink`, there is no need to define it here

Reply:
Indeed it's working without it :) my apologies, I used TS, JSX and css-in-js but not at the same time. Thanks for the feedback.